tests/test_onco_print.py

```diff
@@ -94,3 +120,38 @@ def assert_callback(prop_value, nclicks, input_value):
         component_base=COMPONENT_REACT_BASE,
         data=TEST_DATA
     )
+
+    driver = dash_threaded.driver
+    # assert there is a legend (bar)
+    legend = driver.find_elements_by_class_name('legendbar')
```

Comment:
```suggestion
    legend = wait_for_elements_by_css_selector(driver, '.legendbar')
```
In order to avoid test failure if the component take time to update

Reply:
... if the *prop* took time to update, right?

Oops, I didn't notice the difference between `wait_for_element_by_css_selector()` and `wait_for_elements_by_css_selector()`!!